node/src/BaseClient.ts

```diff
@@ -3438,6 +3439,42 @@ export class BaseClient {
         return this.createWritePromise(createZScore(key, member));
     }
 
+    /**
+     * Computes the union of sorted sets given by the specified `keys` and stores the result in `destination`.
+     * If `destination` already exists, it is overwritten. Otherwise, a new sorted set will be created.
+     * To get the result directly, see {@link zunionWithScores}.
+     *
+     * @see {@link https://valkey.io/commands/zunionstore/|valkey.io} for details.
+     * @remarks When in cluster mode, all keys in `keys` must map to the same hash slot.
```

Comment:
```suggestion
     * @remarks When in cluster mode, `destination` and all keys in `keys` both must map to the same hash slot.
```
Or copy a notice from another command which has `destination` and `keys` (plural)